TESTING.md

```diff
@@ -278,6 +280,129 @@ To run these tests, you go inside `test-artifacts` and execute:
 
 The execution then proceeds as explained in [Binary testing](#Binary-testing).
 
+## Building the test environment
```

Comment:
I would this sections much more to TOP of page. Cause it's prerequisite to run the tests.

Reply:
Good idea. I was also thinking of using the word "prerequisite" in the section title

Reply:
[Updated](https://github.com/dataclouder/terraform-provider-vcd/blob/edge-gateway-creation/TESTING.md)